### PR TITLE
Add style information for TEST_IGNORE()

### PR DESCRIPTION
**Pull request by wolf99**
Clarify usage of `TEST_IGNORE()` for future contributions. This is according to the discussion during PR #179

### CODE REVIEW

File: docs/C_STYLE_GUIDE.md [hunk 1]
```diff
@@ -122,6 +122,29 @@ void tearDown(void)
```
Comment (Gamecock) [inline docs/C_STYLE_GUIDE.md]: In a future we should add a section about helper functions here.  Let me know if you want me to do it.
This PR LGTM.

Comment (wolf99) [reply docs/C_STYLE_GUIDE.md]: I can add it into this PR shortly 👍

File: docs/C_STYLE_GUIDE.md [hunk 2]
```diff
@@ -122,6 +122,29 @@ void tearDown(void)
 
 Next are the test functions themselves. 
 The names of these functions use a `test_` prefix.
+Excepting the first, each test function should have `TEST_IGNORE();` as its first statement. 
+The first occurance of this should be followed by the line comment `// delete this line to run test`.
```
Comment (ryanplusplus) [inline docs/C_STYLE_GUIDE.md]: `occurance` => `occurrence`

**Comment by ryanplusplus** [comment]
Looks good except for a small spelling mistake.

**Comment by wolf99** [comment]
Thanks @ryanplusplus, have corrected that in the latest commit. This should be good to go now

**Pull request merged**